### PR TITLE
3 edited pages ready for review

### PR DESCRIPTION
**Pull request by annieengmark**
The previous six pages had the original curriculum content simply split into the six pages. This pull requests is to update the first three pages with the edited content so you can clearly see the changes.

Please review and add comments, we can discuss before merging if needed.

### CODE REVIEW

File: Pages/Page 1_Intro TOC.md [hunk 1]
```diff
@@ -46,23 +42,45 @@ _insert image of app_
...
+
+
+[leiningen]: https://github.com/technomancy/leiningen
+
+### Running The Server & Entering Commands in the terminal
+
+#### Server
+Confirm your server is running properly. To start a web server for the application, enter the following command in the terminal:
+
+    lein ring server
```
Comment (crkoehnen) [inline Pages/Page 1_Intro TOC.md]: I don't know if this will work yet.  The app hasn't been created.

File: Pages/Page 1_Intro TOC.md [hunk 2]
```diff
@@ -72,5 +90,21 @@ work the same.
     heroku-toolbelt/3.25.0 (x86_64-linux-gnu) ruby/2.1.5
     You have no installed plugins.
 
-With the tools installed, we can begin working on our web app.
+Only enter the first line in the command line; the following lines are the results of your command and indicate the versions of software you have installed on your computer.
```
Comment (crkoehnen) [inline Pages/Page 1_Intro TOC.md:93]: Explaining that is a nice touch.

File: Pages/Page 1_Intro TOC.md [hunk 3]
```diff
@@ -72,5 +90,21 @@ work the same.
...
+## Moving Forward
+Once you confirm your server is working properly, move to the next page where were will briefly go over what Clojure is, what the Web is and how we interact with it, and how HTML works.
+
+Next page, [Chapter 2: Big Picture]
+[Chapter 2: Big Picture]: Page_2_Big_Picture
+
+
+### Copyright and License
+
+Copyright © 2015 Chris Koehnen
```
Comment (crkoehnen) [inline Pages/Page 1_Intro TOC.md:108]: Completely off topic, we should probably look at assigning the copyrights to bridgefoundry or clojurebridge nat.

**Comment by crkoehnen** [comment]
I like it!

File: Pages/Page 1_Intro TOC.md (hunk 1 above)
Comment (crkoehnen) [inline Pages/Page 1_Intro TOC.md]: I double checked and this won't work here.  The student would get an error.

The version check is intact in the "Entering Commands" section.  (Typo in "Comands")  Maybe we could just move the server section later in the tutorial?

File: Pages/Page 1_Intro TOC.md [hunk 4]
```diff
@@ -46,23 +42,45 @@ _insert image of app_
...
+
+#### Server
+Confirm your server is running properly. To start a web server for the application, enter the following command in the terminal:
+
+    lein ring server
 
-    $: lein -version
+This will start the server that allows you to view your app in the web browser. 
+
+#### Entering Comands
```
Comment (crkoehnen) [inline Pages/Page 1_Intro TOC.md:71]: Typo; "comands".

**Comment by annieengmark** [comment]
Closing this PR - it's outdated.

**Pull request closed without merging**